app/assets/javascripts/oxalis/model/sagas/skeletontracing_saga.js

```diff
@@ -202,11 +202,11 @@ function* diffEdges(
 
 function updateTreePredicate(prevTree: TreeType, tree: TreeType): boolean {
   return (
-    prevTree.branchPoints !== tree.branchPoints ||
+    !_.isEqual(prevTree.branchPoints, tree.branchPoints) ||
```

Comment:
Why is this necessary? Why doesn't a referential comparison suffice? If the branchpoints of a tree didn't change, we shouldn't create a new instance?